src/ebonite/build/docker.py

```diff
@@ -80,6 +79,15 @@ class DockerHost(RuntimeEnvironment.Params):
     def __init__(self, host: str = ''):
         self.host = host
 
+    def get_runner(self):
+        """
+        :return: docker runner
+        """
+        if self.default_runner is None:
```

Comment:
We do not initialize `default_runner` in `__init__`. Wouldn't the first call to `get_runner` fail with an `AttributeError`?

Reply:
no, because there is class attribute in `RuntimeEnvironment.Params`